### PR TITLE
introduce docker-images source

### PR DESCRIPTION
**Pull request by muratayusuke**
Add source of `docker images`

### CODE REVIEW

File: sources/docker-images.zsh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+if [ `cat /etc/group | grep docker | grep $USER` ]; then
+    DOCKER_CMD="docker"
+else
+    DOCKER_CMD="sudo docker"
+fi
+
+function zaw-src-docker-images() {
+    desc=`$DOCKER_CMD images 2>/dev/null | awk 'NR!=1{printf"%s/%s [%s] (%s %s %s)\n", $1, $2, $3, $4, $5, $6}'`
+    cand=`echo $desc | sed `
```
Comment (termoshtt) [inline sources/docker-images.zsh]: You seem to forget removing this line.

**Comment by muratayusuke** [comment]
@termoshtt Thank you, I removed debug code.

**Comment by xeno1991** [comment]
I think you should remove `/bin/bash` in `zaw-docker-run` because it may not work in case `ENTRYPOINT` is set.

For example, you can't run the image built from `Dockerfile` below.
```Dockerfile
FROM ubuntu
RUN apt-get install -y zsh
ENTRYPOINT ["/usr/bin/zsh"]
```

**Comment by willghatch** [comment]
As discussed in #68, I think we will be moving to just having links in the readme to more sources.  Do you want to make a repo for this source, @muratayusuke?

**Comment by termoshtt** [comment]
Please create a separate repository to reduce maintenance cost :< as discussed in #68 
I will close this PR. If anyone needs this source, I can create a separate repository. Please let me know.

**Pull request closed without merging**